orchestrator/orchestratorAutomation.ps1

```diff
@@ -35,26 +39,50 @@ Param (
    [string] $storageport
 )
 
-Import-Module 'WebAdministration'
-    if(!$config){   
-        $config = "C:\Program Files (x86)\UiPath\Orchestrator\web.config"
-    }
+Import-Module 'WebAdministration' 
+    
 
-    if(!$aifport){   
-        $aifport = "31390"
-    }
+if(!$config){   
+    $config = "C:\Program Files (x86)\UiPath\Orchestrator"
+} 
 
-    if(!$storageport){
-        $storageport = "31443"
-    }
+#if path does not end with \ add it
+if( $config -notmatch '\\$' ){
+    $config += '\'
+}
 
-    if($domainBasedAccess.Length -gt 0){
-        $domainBasedAccess = $domainBasedAccess.ToString()
-    } else {
-        $domainBasedAccess = "false"
-    }
+$dll_config = $config + 'UiPath.Orchestrator.dll.config'

```

Comment:
Can we check if this file exists and if it has valid permissions ?

Reply:
script is already running using powershell administrator mode, so permission check is not required

Reply:
actually we faced this issue with one of the customer ... he entered wrong path for config and then the script failed ... it took sometime to figure out that path was wrong. 
Lets check for each file and path existence to avoid such cases in future.

Reply:
file check has been added